### PR TITLE
Add telemetry to quickstart

### PR DESCRIPTION
**Pull request by doodlesbykumbi**
### Desired Outcome

*Please describe the desired outcome for this PR.  Said another way, what was
the original request that resulted in these code changes?  Feel free to copy
this information from the connected issue.*

### Implemented Changes

*Describe how the desired outcome above has been achieved with this PR. In
particular, consider:*

- _What's changed? Why were these changes made?_
- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### Connected Issue/Story

Resolves #[relevant GitHub issue(s), e.g. 76]

CyberArk internal issue link: [insert issue ID]()

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: telemetry/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Conjur telemetry quickstart
+
+In order to consume the telemetry data collected by Conjur, you need to setup a Prometheus node to scrape the
+`/metrics` endpoint in Conjur. You can then use the myriad of tools that work on top of Prometheus to visualise the data. 
+
+Here, we provide a quickstart that sets up Prometheus and Graphana to allow you to quickly explore visualisation of the telemetry data collected by Conjur. To be comprehensive in our demostration, this quickstart additionally provides examples of gathering of metrics from services adjacent to Conjur such as Postgres and metrics about the node where Conjur is running.
```
Comment (gl-johnson) [inline telemetry/README.md:6]:
```suggestion
Here, we provide a quickstart that sets up Prometheus and Grafana to allow you to quickly explore visualisation of the telemetry data collected by Conjur. To be comprehensive in our demostration, this quickstart additionally provides examples of gathering of metrics from services adjacent to Conjur such as Postgres and metrics about the node where Conjur is running. 
```

Same typo appears throughout this document.

**Review by gl-johnson — COMMENTED** [review]
Looking good! Left a couple comments.

File: telemetry/README.md (hunk 1 above)
Comment (doodlesbykumbi) [reply telemetry/README.md:6]: Yikes. I wonder if my mind is gravitating towards graphene or something 😅. I'll fix it

File: docker-compose.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ services:
       DATABASE_URL: postgres://postgres@database/postgres
       CONJUR_DATA_KEY:
       CONJUR_AUTHENTICATORS:
+      CONJUR_TELEMETRY_ENABLED: 'true'
```
[the remark this answers is not in the document]
Comment (doodlesbykumbi) [reply docker-compose.yml:60]: Good point. The Docker-compose change slipped through from my test environment. I'll update

File: telemetry/files/prometheus/alert.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+groups:
+  - name: Hardware alerts
```
[the remark this answers is not in the document]
Comment (doodlesbykumbi) [reply telemetry/files/prometheus/alert.yml:2]: Additional config... Prometheus has an alert manager to deal with dispatching alerts based on rules, see https://prometheus.io/docs/alerting/latest/alertmanager/

**Review by gl-johnson — DISMISSED** [review]
LGTM!

**Comment by szh** [comment]
@gl-johnson and @doodlesbykumbi, any idea why this wasn't merged?

**Review by gl-johnson — APPROVED** [review]

**Pull request merged**